common/presets.ts

```diff
@@ -195,6 +196,7 @@ export const serviceGenMap: Record<Exclude<ChatAdapter, 'default'>, GenMap> = {
     presencePenalty: 'presence_penalty',
     gaslight: 'gaslight',
     oaiModel: 'oaiModel',
+    streamResponse: 'stream',
```

Comment:
`stream` is the field the API expects, though it's not clear that the OAI adapter actually uses the serviceGenMap since it's just building the POST request body at the top of the `handleOAI` function anyway. I mapped it just in case though.

Reply:
yeah this seems unused